src/base/utils/net.h

```diff
@@ -28,13 +28,23 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
 class QString;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+
+        template <typename T>
+        T unquote(const T &str)
```

Comment:
1. Or you may suggest a better name than `unquote`.
  Currently it unquotes double quotation marks and has a unexpected behavior as you see, thus it's only suitable for http/net use.

2. Reason I kept `Utils::Net::unquote` is because it's unique behavior is actually a common pattern that I found in `RequestParser` (that I might refactor later).